src/socketify/loop.py

```diff
@@ -89,10 +89,10 @@ def _keep_alive(self):
             self.is_idle = True
                 
             if relax:
-                self.uv_loop.run_nowait()
+                self.uv_loop.run_once()
                 self.loop.call_later(0.001, self._keep_alive)
             else:
-                self.uv_loop.run_nowait()
+                self.uv_loop.run_once()
```

Comment:
run_once will idle hang when nothing is todo in uv loop, but we may have things to do in asyncio loop, like in graphQL case